src/components/types.ts

```diff
@@ -286,3 +286,31 @@ export interface PDFViewerProps {
   /** The url of the PDF file to be displayed. */
   url: string
 }
+
+export interface Weather {
+  timestamp: number
+  temp: {
+    low: number
+    high: number
+    current?: number
+  }
+  weatherIcon: {
+    icon: string
+    description: string
+  }
+}
+
+export interface WeatherFormat extends DataFormat {
+  /** Array of daily weather data. */
+  weather: Weather[]
```

Comment:
Does this only support day wise breakdown? What about hourly breakdown?

Reply:
At the moment it is by the day. What do you think?

Reply:
Ok. Please ensure that's mentioned clearly in the component description